src/Stoa.ts

```diff
@@ -828,6 +829,55 @@ class Stoa extends WebService
             );
     }
 
+    /**
+     * GET /wallet/blocks/header
+     *
+     * Called when a request is received through the `/wallet/blocks/header`
+     *
+     * Returns the information abort the block header.
+     */
+    private getWalletBlocksHeader (req: express.Request, res: express.Response)
+    {
+        if ((req.query.height !== undefined) &&
+            !Utils.isPositiveInteger(req.query.height.toString()))
+        {
+            res.status(400).send(`Invalid value for parameter 'height': ${req.query.height.toString()}`);
+            return;
+        }
+
+        let height = (req.query.height !== undefined)
+            ? new Height(req.query.height.toString())
+            : null;
+
+        if (height != null)
+            logger.http(`GET /wallet/blocks/header height=${height.toString()}`);
+        else
+            logger.http(`GET /wallet/blocks/header`);
+
+        this.ledger_storage.getWalletBlocksHeaderInfo(height)
+            .then((rows: any[]) =>
+            {
+                if (!rows.length)
+                {
+                    res.status(204).send(`No blocks`);
+                    return;
+                }
+
+                let info = {
+                    height: rows[0].height.toString(),
+                    hash: new Hash(rows[0].hash, Endian.Little).toString(),
+                    merkle_root: new Hash(rows[0].merkle_root, Endian.Little).toString(),
+                    time_stamp: rows[0].time_stamp
+                };
```

Comment:
It does not contain all the information in the `block header` like the function name`getWalletBlocksHeader`.

Reply:
> It does not contain all the information in the `block header` like the function name`getWalletBlocksHeader`.

I will upgrade after random_seed, missing_validators, and merkle_tree are added to the database.
Currently, I only put what will be used in agora wallet.